negotiation/neural/preprocess.py

```diff
@@ -216,18 +216,18 @@ def _add_utterance(self, agent, utterance, lf=None):
         else:
             lf = []
 
-        if not new_turn:
-            self.token_turns[-1].extend(utterance)
-            self.lfs[-1].extend(lf)
-            self.entities[-1].extend(entities)
-        else:
+        if new_turn:
```

Comment:
why did you remove the if not new_turn part?

Reply:
There is still a "new_turn" check, I just reversed the order to avoid the "not"
Since new_turn is True or False, one of those if-else blocks must run, so it should be safe.  Let me know if that's not true.